src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Added `ParentId` to `TelemetrySpan` ([#2733](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2733))_
```

Comment:
```suggestion
* Added `ParentId` to `TelemetrySpan` ([#2740](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2740/files))_
```

This should be a link to the pull request, which unfortunately you don't actually get until you make the PR and are given the ID and then have to come back and update the changelog 😅